runtime/runtime_test.go

```diff
@@ -0,0 +1,184 @@
+package runtime
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"syscall"
+	"testing"
+	"time"
+
+	utils "github.com/docker/containerd/testutils"
+)
+
+var (
+	referenceSpec = "config.json"
+	devNull       = "/dev/null"
+	stdin         io.WriteCloser
+)
+
+func generateReferenceSpecs(destination string) error {
```

Comment:
Could you move this to your util package as a public method to avoid duplication with integration-test (it's called  in `integration-test/check_test.go`?